modules/ROOT/partials/configuration/remove_trailing_brs.adoc

```diff
@@ -1,6 +1,9 @@
 [[remove_trailing_brs]]
 == `+remove_trailing_brs+`
 
+[IMPORTANT]
+This option has been deprecated in {productname} 7.0.
```

Comment:
`remove_trailing_brs` was already deprecated with `TINY-10454` this function has been removed

Reply:
@lorenzo-pomili yes, but the functions fix-version is set to 7.0? And we have not documented the removal of this option. So we still need to add it to the release-notes for 7.0, and insert it to the `migration-guide` for v6 to v7 users as a deprecated option.

Reply:
If the option has been removed, then perhaps `modules/ROOT/partials/configuration/remove_trailing_brs.adoc` file could be deleted altogether in TinyMCE 7.0 Docs.

Reply:
@lorenzo-pomili good pickup, I'll update accordingly, sorry for the confusion.

Reply:
@lorenzo-pomili As discussed, ive `deleted` all reference to `remove_trailing_brs` from the `7.0` docs project, and have added a checkbox for use to ensure this is noted in the `migration guide`.